docs/basics/networkvariable.md

```diff
@@ -21,6 +21,9 @@ A `NetworkVariable`:
 - Property *must* be defined within a `NetworkBehaviour` derived class attached to a `GameObject`
     - The `GameObject` or a parent `GameObject` **must** also have a `NetworkObject` component attached to it.
 - A `NetworkVariable`'s assigned type (`T`) must be [constrained to an unmanaged `Type`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/constraints-on-type-parameters#unmanaged-constraint).
+- A `NetworkVariable`'s value can only be set when:
+    - Initializing the property (either when it is declared or within the Awake method)
+    - When the associated `NetworkObject` is spawned.
```

Comment:
Can we go with "**After** the associated `NetworkObject` is spawned" ? Just to be sure it is not limited to the moment at which it spawns, but also allowed afterwards

Reply:
How about "While the associated `NetworkObject` is spawned"?